src/main/java/io/orten/nano/impl/API.java

```diff
@@ -0,0 +1,25 @@
+package io.orten.nano.impl;
+
+import io.orten.nano.model.Organization;
+import io.orten.nano.util.Database;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Response;
+
+
```

Comment:
remove duplicate empty line

Reply:
This API file is extra so it is removed now